cluster_test.go

```diff
@@ -1693,20 +1693,47 @@ func (s *S) TestMaxSocketUses(c *C) {
 		time.Sleep(5e8)
 	}
 
-	// a socket was used during connection
-	numUses := 1
+	for i := 0; i < 2; i++ {
+		stats = mgo.GetStats()
+		// every 4th usage should result in a new connection
+		c.Logf("Active connections: %d", stats.SocketsAlive)
+		s := session.Copy()
+		defer s.Close()
+		c.Check(s.Ping(), IsNil)
+		time.Sleep(time.Second)
+		s.Refresh()
+	}
 
-	// acquire and release sockets, the pool should grow to minPoolSize
-	for i := 0; i < 24; i++ {
+	c.Assert(stats.SocketsExpired, Equals, 1)
+}
+
+func (s *S) TestNoMaxSocketUses(c *C) {
```

Comment:
Is this correctly named since it's testing connetion life time, and no longer number of uses?